gateway/webhook/webhookTransformer.go

```diff
@@ -45,12 +45,14 @@ func (bt *batchWebhookTransformerT) markResponseFail(reason string) transformerR
 	return resp
 }
 
-func (bt *batchWebhookTransformerT) transform(events [][]byte, sourceType string) transformerBatchResponseT {
+func (bt *batchWebhookTransformerT) transform(events [][]byte, payloadArr [][]byte, sourceType string, version string) transformerBatchResponseT {
 	bt.stats.sentStat.Count(len(events))
 	transformStart := time.Now()
 
 	payload := misc.MakeJSONArray(events)
-	url := fmt.Sprintf(`%s/%s`, bt.sourceTransformerURL, strings.ToLower(sourceType))
+	sourceTransformURL := strings.Replace(bt.sourceTransformerURL, "{version}", version, -1)
```

Comment:
change `bt.sourceTransformerURL` to v1 where its initialised

Reply:
No need of strings replace

Reply:
Done